<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>History</class>
 <widget class="QDialog" name="History">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>531</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
    </font>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>文本</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>380</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>最近历史记录</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>421</height>
    </rect>
   </property>
   <property name="text">
    <string> </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
